src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = days * 40;
+
+  if (days >= 7) {
+    amount -= 50;
```

Comment:
According to the checklist, you should return the value immediately in if statements.